extensions/cms/src/cmsResource/tree/treeProvider.ts

```diff
@@ -35,18 +35,18 @@ export class CmsResourceTreeProvider implements TreeDataProvider<TreeNode>, ICms
 				// Call to collect all locally saved CMS servers
 				// to determine whether the system has been initialized.
```

Comment:
So why is it necessary to do this check here inside getChildren? Shouldn't you load the servers from the configuration once when the extension is loaded and then getChildren can just use the in-memory cache?

Reply:
No, we wanted the updated CMS server, in case someone changed it in SSMS for example.